helm/cas-ciip-portal/templates/deploy.yaml

```
@@ -61,6 +61,8 @@ spec:
             value: {{ .Values.env.adminEmail }}
           - name: FEEDBACK_SITE_URL
             value: {{ .Values.fider.host }}
+          - name: ENABLE_ANALYTICS
+            value: {{ .Values.analytics.enable }}
```

k8s environment variables have to be strings
```suggestion
            value: {{ .Values.analytics.enable | quote }}
```